src/OrchardCore/OrchardCore.DisplayManagement.Liquid/Filters/ShapeStringifyFilter.cs

```diff
@@ -38,17 +34,12 @@ static async ValueTask<FluidValue> Awaited(Task<IHtmlContent> task)
                     return Awaited(task);
                 }
 
-                StringValue value;
-                using (var writer = new StringWriter())
-                {
-                    task.Result.WriteTo(writer, NullHtmlEncoder.Default);
-                    value = new StringValue(writer.ToString(), false);
-                }
-
-                return new ValueTask<FluidValue>(value);
+                using var writer = new StringWriter();
```

Comment:
If it is not written to the page output directly it doesn't need to be buffered anymore since you are assume a global buffer for liquid now. Or is it not just liquid here?

Reply:
It is the `shape_stringify` filter, not the `shape_render`, it is not intended to be written to the buffer, but e.g. to assign a local liquid var. It was already using a `StringWriter` and just did here a little refactoring

And above in the `Awaited()` method it was using an `HtmlContentValue` because i assume a bad copy / paste from the `ShapeRenderFilter`, so i updated it to do the same

Reply:
It is rarely used compared to the shape render filter